pages/textbook-beta/summer-school/quantum-computing-and-quantum-learning-2021.vue

```diff
@@ -0,0 +1,315 @@
+
+      headerImg = '/images/textbook-beta/summer-school/quantum-computing-and-quantum-learning-2021/header.png'
+
+  backToTextbookHomeLink = {
+    url: '/textbook-beta',
+    label: 'Home',
+    segment: {
+      cta: 'back-to-textbook-home', location: 'header'
+    }
+  }
+
+  startLearningCTA: GeneralLink = {
+    url: summerSchoolCourseStartLearningUrl,
+    label: 'Start learning',
+    segment: {
+      cta: 'introduction', location: 'header'
+    }
+  }
+
+  links: GeneralLink[] = [
+    {
+      url: 'http://mmrc.amss.cas.cn/tlb/201702/W020170224608149940643.pdf',
+      label: 'Quantum Computation and Quantum Information by Nielsen & Chuang',
+      segment: {
+        cta: 'qc-qi-pdf',
+        location: 'external-recommended-readings'
+      }
+    },
+    {
+      url: 'https://www.deeplearningbook.org/',
+      label: 'Deep Learning by Ian Goodfellow et al',
+      segment: {
+        cta: 'deep-learning',
+        location: 'external-recommended-readings'
+      }
+    }
+  ]
+
+  courses: Course[] = [
+    {
+      image: 'QGSS2021_Lecture1.1_CoverImage.png',
+      label: 'Vector Spaces, Tensor Products, and Qubits',
+      segment: { cta: 'lecture-1-1', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lec1-1'
+    },
+    {
+      image: 'QGSS2021_Lecture1.2_CoverImage.png',
+      label: 'Introduction to Quantum Circuits',
+      segment: { cta: 'lecture-1-2', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lec1-2'
+    },
+    {
+      image: 'QGSS2021_Lecture2.1_CoverImage.png',
+      label: 'Simple Quantum Algorithms I',
+      segment: { cta: 'lecture-2-1', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lec2-1'
+    },
+    {
+      image: 'QGSS2021_Lecture2.2_CoverImage.png',
+      label: 'Simple Quantum Algorithms II',
+      segment: { cta: 'lecture-2-2', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lec2-2'
+    },
+    {
+      image: 'QGSS2021_Lecture3.1_CoverImage.png',
+      label: 'Noise in Quantum Computers pt 1',
+      segment: { cta: 'lecture-3-1', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lec3-1'
+    },
+    {
+      image: 'QGSS2021_Lecture3.2_CoverImage.png',
+      label: 'Noise in Quantum Computers pt. 2',
+      segment: { cta: 'lecture-3-2', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lec3-2'
+    },
+    {
+      image: 'QGSS2021_Lab1_CoverImage.png',
+      label: 'Lab 1: Quantum Computing Algorithms and Operations',
+      segment: { cta: 'lab-1', location: 'summer-school-2021' },
+      url: 'https://learn.qiskit.org/course/qgss2021/lab1'
+    },
+    // {
```

Comment:
If we are not going to need this, you can remove it

Reply:
removed!